components/ListClaims.vue

```diff
@@ -36,7 +36,7 @@ export default {
 
 <style>
   .claims-text {
```

Comment:
This css should probably go into main.css or custom.css

Reply:
Done